src/pydata_sphinx_theme/assets/styles/sections/_header.scss

```diff
@@ -100,3 +109,25 @@
   padding-left: 2px;
   vertical-align: middle;
 }
+
+label.primary-toggle {
+  margin: 0 1em;
+  position: absolute;
```

Comment:
Note we have some redundant CSS in here - why not define another class that both share, and then split the CSS between that shared class and the more specific class. E.g. `<label class="drawer-toggle primary-toggle">`